scripts/sanitycheck

```diff
@@ -3317,6 +3322,16 @@ def main():
         error("west-flash requires device-testing to be enabled")
         sys.exit(1)
 
+    if os.name == 'nt' and \
+       (not options.build_only or not options.disable_size_report):
+        print("Testing with sanitycheck on Windows is not fully supported yet.")
+        print("Only build without size report works for now:")
+        print("\t- '--build-only --disable-size-report'")
```

Comment:
I don't think burying the start of Windows documentation in just one catch-all Windows error message is a good idea. I think it's better and more "scalable" to keep Windows-specific tweaks separate from each other and generally closer to the corresponding code. This also allows "smarter" behaviour, here for instance.

```python
if os.name == 'nt' and not options.disable_size_report:
     warning ("Windows doesn't support size report yet, forcing --disable-size-report")
     options.disable_size_report = true
```
Just a suggestion/example of something not really possible with a grab bag error message.